tests/test_exceptions.py

```diff
@@ -21,16 +21,24 @@
 
 
 class TestUtils(unittest.TestCase):
+    maxDiff = None
```

Comment:
[nits]
it seems unused

Reply:
This is a special attribute of `unittest.TestCase`